flink-cdc-connect/flink-cdc-source-connectors/flink-connector-debezium/src/main/java/com/ververica/cdc/debezium/event/DebeziumEventDeserializationSchema.java

```diff
@@ -0,0 +1,537 @@
+    }
+
+    private RecordData extractDataRecord(Struct value, Schema valueSchema) throws Exception {
+        DataType dataType = ConnectSchemaTypeInference.infer(valueSchema);
+        return (RecordData)
+                getOrCreateConverter(dataType, serverTimeZone).convert(value, valueSchema);
+    }
+
+    private static DeserializationRuntimeConverter getOrCreateConverter(
+            DataType type, ZoneId serverTimeZone) {
+        return CONVERTERS.computeIfAbsent(type, t -> createConverter(t, serverTimeZone));
+    }
+
+    // -------------------------------------------------------------------------------------
+    // Runtime Converters
+    // -------------------------------------------------------------------------------------
+
+    /** Creates a runtime converter which is null safe. */
+    private static DeserializationRuntimeConverter createConverter(
+            DataType type, ZoneId serverTimeZone) {
+        return wrapIntoNullableConverter(createNotNullConverter(type, serverTimeZone));
+    }
+
+    // --------------------------------------------------------------------------------
+    // IMPORTANT! We use anonymous classes instead of lambdas for a reason here. It is
+    // necessary because the maven shade plugin cannot relocate classes in
+    // SerializedLambdas (MSHADE-260).
+    // --------------------------------------------------------------------------------
+
+    /** Creates a runtime converter which assuming input object is not null. */
+    public static DeserializationRuntimeConverter createNotNullConverter(
+            DataType type, ZoneId serverTimeZone) {
+        // if no matched user defined converter, fallback to the default converter
+        switch (type.getTypeRoot()) {
+            case BOOLEAN:
+                return convertToBoolean();
+            case TINYINT:
+                return new DeserializationRuntimeConverter() {
+
+                    private static final long serialVersionUID = 1L;
+
+                    @Override
+                    public Object convert(Object dbzObj, Schema schema) {
+                        return Byte.parseByte(dbzObj.toString());
+                    }
+                };
+            case SMALLINT:
+                return new DeserializationRuntimeConverter() {
+
+                    private static final long serialVersionUID = 1L;
+
+                    @Override
+                    public Object convert(Object dbzObj, Schema schema) {
+                        return Short.parseShort(dbzObj.toString());
+                    }
+                };
+            case INTEGER:
+                return convertToInt();
+            case BIGINT:
+                return convertToLong();
+            case DATE:
+                return convertToDate();
+            case TIME_WITHOUT_TIME_ZONE:
+                return convertToTime();
+            case TIMESTAMP_WITHOUT_TIME_ZONE:
+                return convertToTimestamp(serverTimeZone);
```

Comment:
After we support convert mysql timestamp to our timestamp_ltz, we do not need serverTimezone anymore for mysql datetime ?